test/built-ins/Promise/race/resolved-sequence-mixed.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-promise.race
+description: >
+  Resolution ticks are set in a predictable sequence of mixed fulfilled and rejected promises
+info: |
+  PerformPromiseRace
+
+  Repeat,
+    Let next be IteratorStep(iteratorRecord).
+    If next is an abrupt completion, set iteratorRecord.[[Done]] to true.
+    ReturnIfAbrupt(next).
+    If next is false, then
+      Set iteratorRecord.[[Done]] to true.
+      Return resultCapability.[[Promise]].
+    Let nextValue be IteratorValue(next).
+    If nextValue is an abrupt completion, set iteratorRecord.[[Done]] to true.
+    ReturnIfAbrupt(nextValue).
+    Let nextPromise be ? Call(promiseResolve, constructor, « nextValue »).
+    Perform ? Invoke(nextPromise, "then", « resultCapability.[[Resolve]], resultCapability.[[Reject]] »).
+
+flags: [async]
+includes: [compareArray.js,promiseHelper.js]
+---*/
+
+let a = new Promise((_, reject) => reject(''));
```

Comment:
(Here and elsewhere): Wouldn't this be equivalent:

let a = Promise.reject('');

Or is there a subtle difference I'm missing?